client/reader/discover/controller.js

```diff
@@ -8,14 +8,36 @@ import {
 	trackScrollPage,
 } from 'calypso/reader/controller-helper';
 import { recordTrack } from 'calypso/reader/stats';
+import PostPlaceholder from 'calypso/reader/stream/post-placeholder';
 import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
 import getCurrentQueryArguments from 'calypso/state/selectors/get-current-query-arguments';
 import getCurrentRoute from 'calypso/state/selectors/get-current-route';
 import renderHeaderSection from '../lib/header-section';
+import { DiscoverHeader } from './discover-stream';
 import { getSelectedTabTitle, DEFAULT_TAB } from './helper';
 
 const ANALYTICS_PAGE_TITLE = 'Reader';
 
+const DiscoverPageDocumentHead = ( { tabTitle } ) => {
+	const translate = useTranslate();
+
+	const title = translate( 'Browse %s blogs & read articles ‹ Reader', {
+		args: [ tabTitle ],
```

Comment:
Not related to the changes in this PR, but I just noticed that the `tabTitle` argument will have [a hardcoded untranslated value](https://github.com/Automattic/wp-calypso/blob/c9467f7afcc777c2850861e67f9c8bf10cdbf2c5/client/reader/discover/helper.js#L25).

Reply:
Thanks, I saw that too! We'll follow up on this.